new_repository.md

```diff
@@ -319,10 +319,55 @@ com/populationgenomics/team-docs/main/new_repository/github-workflows-condarse.y
  -O .github/workflows/condarse.yaml
 ```
 
-Now, whenever you run `git push --tags`, a `Condarise` GitHub workflow will be triggered,
-build the package versioned after the corresponding git tag, and upload it to the
-Anacodna CPG channel. After that, the package will be available with:
+Now, whenever you run `git push --tags`, a `Condarise` GitHub workflow will be
+triggered, build the package versioned after the corresponding git tag, and upload it to
+the Anacodna CPG channel. After that, the package will be available with:
 
 ```bash
 conda install -c cpg -c conda-forge <my-project>
 ```
+
+## Making a release
+
+In order to release a new version of a tool that is shipped with conda, you would need
+to bump version with `bump2version` and push the created git tag, and also push the "
+Bump version" commit. Because direct pushes to the `main` branch are disabled by
```

Comment:
```suggestion
to bump version with `bump2version` and push the created git tag, and also push the
"Bump version" commit. Because direct pushes to the `main` branch are disabled by
```